treatment-service/services/message_queue_service.go

```diff
@@ -0,0 +1,59 @@
+package services
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/caraml-dev/xp/treatment-service/config"
+	"github.com/caraml-dev/xp/treatment-service/models"
+)
+
+type MessageQueueService interface {
```

Comment:
Similar comment as with the management service. I think we should put the existing `subscription.go` and the new NoopMQ implementation in separate files, under a `messagequeue` package. Also the `ExperimentSubscriber ` interface and be removed.